src/api/flaskr/service/order/funs.py

```
@@ -177,6 +196,22 @@ def send_order_feishu(app: Flask, record_id: str):
     send_notify(app, title, msgs)
 
 
+def send_revoke_feishu(app: Flask, order_bid: str, user_identify: str):
+    with app.app_context():
+        order: Order = Order.query.filter(Order.order_bid == order_bid).first()
+        if not order:
+            return
+        shifu_info: LearnShifuInfoDTO = get_shifu_info(app, order.shifu_bid, False)
+        title = "取消课程授权通知"
+        msgs = [
+            "用户标识：{}".format(user_identify),
```

Ruff (0.15.10)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)